lib/base/exception.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include "base/exception.hpp"
 #include <boost/thread/tss.hpp>
+#include <utility>
```

Comment:
For what is this import needed?

Reply:
This is what I needed for the ``std::move()``. Do we have our own move utility?

Reply:
Ah, that's coming from there. Then never mind.